### PR TITLE
Calculator UI is added to the project and and custom components is created for Calculator app

### PR DESCRIPTION
**Pull request by MMhaidari**
# what is added new
- [ ] A new directory is added by the name of Components
- [ ] New files added by the name of Calculator.js and Calculator.css
- [ ] design added that matches the requirement 
- [ ] design only implemented in (JSX and CSS)
- [ ] functional component is used
- [ ] one child component is created
- [ ] project runs without any error in the browser

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      <div className="calculator">
+        <div className="display">0</div>
+        <div className="operator-numbers">
+          <div className="digits">{numbersBtn}</div>
+          <div className="operators">{operatorBtn}</div>
+        </div>
+      </div>
+    </div>
+  );
+};
```
Comment (danmainah) [inline src/components/Calculator.js:32]: - [ ] @MMhaidari great work on your calculator component loved the simplicity, however, you forgot to create a child component to your Calculator component, you are supposed to create a child component before adding the component to top-level component `App.js`
image

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @MMhaidari , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Gitflow was correctly used
✔️  Good design
✔️  Good PR and Description


# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/components/Calculator.js (hunk 1 above)
Comment (MMhaidari) [reply src/components/Calculator.js:32]: @danmainah dear reviewer the CalculatorUI itself is a child component which is passed to the upper-level component which is Calculator function

Comment (danmainah) [reply src/components/Calculator.js:32]: - [ ] @MMhaidari if you check the hierarchy of the instructions, you will discover you are creating a child component after creating a calculator component, and as I have highlighted before you can export the calculator component the child component needs to be have been created, meaning it belongs to the calculator component.Hope its clear now

**Comment by medaminedev66** [comment]
Hello @MMhaidari👋,

_This is an additional review as a result of the second opinion request._

I can see that you have used a child component in the Calculator component. Great job.

I have an optional change that can make your project more organized. In React, it is recommended to create each component in a separate file. This is because React components are reusable and independent units of code that represent a specific part of the UI. By separating each component into its own file, it becomes easier to manage and maintain your codebase as your application grows.

Separating each component into its own file also makes it easier to share and reuse components between different parts of your application, or even between different applications. It also makes it easier for other developers to understand and work with your code.

As this was the only reason that your project was not approved it will be approved as soon as possible.
Please ask for another review in your Dashboard and the next Code Reviewer will mark it as approve right away :)

Keep rocking 🚀!

**Comment by MMhaidari** [comment]
# what is added new
- [ ]  A new directory is added by the name of Components
- [ ]  New files added by the name of Calculator.js and Calculator.css
- [ ]  design added that matches the requirement
- [ ]  design only implemented in (JSX and CSS)
- [ ]  functional component is used
- [ ]  one child component is created
- [ ]  project runs without any error in the browser

**Review by pinkmoon25 — APPROVED** [review]
Hi @MMhaidari , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
Linter checks :heavy_check_mark: 
Gitflow :heavy_check_mark: 
Remove boilerplate :heavy_check_mark: 
Calculator component and design :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**